hw/ip/otbn/dv/otbnsim/stepped.py

```diff
@@ -310,12 +323,16 @@ def on_invalidate_imem(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
     return None
 
 
-def on_reset(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
-    if args:
-        raise ValueError('reset expects zero arguments. Got {}.'
-                         .format(args))
+def on_step_crc(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
+    check_arg_count('step_crc', 2, args)
 
-    return OTBNSim()
+    item = read_word('item', args[0], 48)
+    state = read_word('state', args[1], 32)
+
+    new_state = binascii.crc32(item.to_bytes(6, 'little'), state)
+    print(f'! otbn.LOAD_CHECKSUM: 0x{new_state:08x}')
```

Comment:
I guess this print is harmless, right?

Reply:
It's actually needed! This is how we signal the new version of the checksum back to the C++ code.